tests/test_engine.py

```diff
@@ -2009,8 +2019,7 @@ def my_task(x):
 
         assert (
             "Received non-final state 'Failed' when proposing final state 'Failed' and"
-            " will not attempt to run again..."
-            not in caplog.text
+            " will not attempt to run again..." not in caplog.text
```

Comment:
The static analysis stuff will probably fail with this. `prefect` uses `ruff` now which has some minor inconsistencies with black formatting.

Reply:
yup on it thanks 👍 

thought i fixed these already haha